bundle/Controller/ContentEditController.php

```diff
@@ -46,7 +48,19 @@ public function __construct(
      *
      * @return \EzSystems\RepositoryForms\Content\View\ContentCreateView|\Symfony\Component\HttpFoundation\Response
```

Comment:
is this valid @return doc?

Reply:
I think yes. Documentation says 'Function can return either of two types'
` /**
  * @return string|null The label's text or null if none provided.
  */
`

Reply:
Not my point :)

Reply:
Fixed @ViniTou